optimum/habana/diffusers/schedulers/scheduling_euler_discrete.py

```diff
@@ -257,7 +277,10 @@ def step(
         # 2. Convert to an ODE derivative
         derivative = (sample - pred_original_sample) / sigma_hat
 
-        dt = sigma_next - sigma_hat
+        if self.is_inference:
```

Comment:
can we try to remove this flag , keep it as default implementation/optimal implementation, unless it affects any model